src/containers/guest-home-page/login-form/LoginForm.jsx

```diff
@@ -31,6 +33,8 @@ const LoginForm = ({
     openModal({ component: <ForgotPassword /> })
   }
 
+  const inputsAreValid = useMemo(() => data.email && data.password)
```

Comment:
useMemo without dependency does nothing, and as I see we don`t need even it in this code. so you can just put this code in button, disabled={!data.email || !data.password}

Reply:
You are right, it will be much better that way. Sorry for the silly mistakes, I still have a lot to learn 😅. Fixed it